Assets/Scripts/SS3D/Systems/Health/DamageType.cs

```diff
@@ -0,0 +1,14 @@
+public enum DamageType
+{
+    Crush,
+    Slash,
+    Puncture,
+    Pressure,
+    Heat,
+    Cold,
+    Shock,
+    Rad,
+    Toxic,
+    Acid,
+    Oxy,
```

Comment:
"Respiratory" maybe? I don't feel its correct to use any simplifications like "Oxy", could lead to future confusion

Reply:
Oxy is what ss13 uses for basically the same thing so I dont think there will be much confusion there.

But there could be confusion calling it respiratory because of the respiratory system (body layer) the character has.